ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -207,61 +176,49 @@ public void onInitializeSuccess() {
           public void onInitializeError(String errorMessage) {
             Log.d(TAG, "SDK init failed: " + VungleBannerAdapter.this);
             VungleManager.getInstance().removeActiveBannerAd(mPlacementId);
-            VungleListener listener = mVungleListener.get();
-            if (mPendingRequestBanner && listener != null) {
-              listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
+            MediationBannerListener listener = bannerListener.get();
+            if (mPendingRequestBanner && listener != null && adapter.get() != null) {
+              listener.onAdFailedToLoad(adapter.get(), AdRequest.ERROR_CODE_INTERNAL_ERROR);
```

Comment:
I would propose to have helper ststic function onAdFailedToLoad(code) which could help to eliminate all this "if"s

Reply:
done.